docs/content/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Apache Flink Table Store
+type: docs
+bookToc: false
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Apache Flink Table Store
+
+Flink Table Store is a unified streaming and batch store for building dynamic
+tables on Apache Flink. It is designed to be the best connector to Flink as
+the storage for streaming warehouse. It uses a full Log-Structured Merge-Tree
+(LSM) structure for high speed and large amount of data update & query capability.
+
+The Table Store supports the following usage:
+- **Streaming Insert**: Write changelog streams, including cdc from database and streams from Flink Sink.
```

Comment:
from Flink Sink? should it either be 
`Write changelog streams, including CDC from database and streams from Flink Source` 
or 
`Write changelog streams, including CDC from database and streams to Flink Sink`

p.s. `cdc`better be capitalized

Reply:
Let's remove `from Flink Sink`